lib/open_api_spex/cast/string.ex

```diff
@@ -30,6 +30,16 @@ defmodule OpenApiSpex.Cast.String do
     end
   end
 
+  def cast(%{value: value, schema: %{format: :byte}} = ctx) when is_binary(value) do
+    case Base.decode64(value) do
+      {:ok, result} ->
```

Comment:
[suggestion]

```elixir
case Base.decode64(value) do
 {:ok, _} = result -> 
   result
   _ ->
   Cast.error(ctx, {:invalid_format, :base64})
end
```

Returning the existing tuple will be faster.

```elixir
Benchee.run(
  %{
    "old_result" => fn ->
      case fun.() do
        {:ok, _} = res -> res
        _ -> :error
      end
    end,
    "new_result" => fn ->
      case fun.() do
        {:ok, res} -> {:ok, res}
        _ -> :error
      end
    end
  },
  time: 2,
  memory_time: 2
)
```

```
Name                 ips        average  deviation         median         99th %
old_result        2.15 M      464.57 ns  ±4255.55%         375 ns         562 ns
new_result        1.48 M      673.76 ns  ±2837.16%         542 ns         750 ns

Comparison: 
old_result        2.15 M
new_result        1.48 M - 1.45x slower +209.19 ns

Memory usage statistics:

Name          Memory usage
old_result         1.18 KB
new_result         1.70 KB - 1.44x memory usage +0.52 KB
```

Reply:
@zorbash fixed & force-pushed the change, thank you!